de_workloads/ingest/Ingest_AzureSql_Example/data_factory/pipelines/Ingest_AzureSql_Example.json

```diff
@@ -222,7 +222,7 @@
                         "typeProperties": {
                             "variableName": "run_id",
                             "value": {
-                                "value": "@{if(empty(pipeline().parameters.run_id), pipeline().RunId, pipeline().parameters.run_id)}",
+                                "value": "@{if(or(empty(pipeline().parameters.run_id),equals(pipeline().parameters.run_id, 'auto')), pipeline().RunId, pipeline().parameters.run_id)}",
```

Comment:
Why this change? Won't it affect how the "automated_test_flag" is calculated in this pipeline?